wandb/apis/public.py

```diff
@@ -452,7 +454,7 @@ def update(self):
         %s
         ''' % RUN_FRAGMENT)
         res = self._exec(mutation, id=self.id, tags=self.tags,
-                         description=self.description, config=self.json_config)
+                         description=self.description, notes=self.notes, display_name=self.display_name, config=self.json_config)
```

Comment:
It looks like self.display_name is never set anywhere. I think this breaks the public API. Maybe that means no one uses it? In any case, will try to fix this in my upcoming CLI PR.

Reply:
There's a magical getter that should grab it from `self._attrs` I believe...